### PR TITLE
enh: sync README between `pt-br` and `eng` repos

### PR DESCRIPTION
**Pull request by paulolimac**
What
====

- Mirroring translations between pt-br and eng repos on `README.asciidoc`.

Why
===

- Pt-br repo have more lines, more text than eng repo.
- Needs to be the same number of sentences.
- Review some translations.

Where
=====

On `./README.asc`.

How
===

Steps to translate the pt-br README:

1. Use dicts, like Google translator and the command line trans.
2. Open all READMEs in paralel:
    - [English readme](https://github.com/progit/progit2/blob/master/README.asc);
    - [Old Pt-br readme](https://github.com/progit/progit2-pt-br/blob/master/README.asc);
    - My Pt-br readme, to be edited;
3. Use the online editor [asciidoclive](https://asciidoclive.com).

Results
=======

Changed `./README.asc` in many sentences.

Future work
===========

- What is a good tool to edit asciidoc (with viewer)?
- Change all markdown files to ascdoc files.
- How to know what sentences need to be translated?
- How to sync between foreign and english repositories?

---

### CODE REVIEW

**Comment by paulolimac** [comment]
indirectly address progit/progit2#1266

**Comment by carlosschults** [comment]
@paulolimac Hi! I'm sorry for the delay, for some reason this has slipped my attention. I'm going to try and merge today or tomorrow.

Thanks for your help!

File: README.asc [hunk 1]
```diff
@@ -1,32 +1,22 @@
...
-O mais fácil é simplesmente deixar que a gente faça isso.
-Um robô fica esperando por mudanças no branch principal e automaticamente gera o livro para todos.
-
-Você acha as versões atuais em http://git-scm.com/book[] e mais informações sobre a geração dos livros estão disponíveis em https://progit.org[].
-
-=== Manualmente
-
-Outra forma de gerar os arquivos do ebooks é de forma manual, usando o Asciidoctor.
-Se você executar os comandos a seguir você _deve_ obter os arquivos em HTML, Epub, Mobi e PDF:
+Você pode gerar os arquivos de e-book manualmente com o Asciidoctor. Se você executar o seguinte, você _pode_ atualmente obter arquivos de saída em HTML, Epub, Mobi e PDF:
```
Comment (carlosschults) [inline README.asc]: The correct translation of "actually" to pt-br would be something like "realmente", or "na verdade." Translating that to "atualmente" is a case of [false friends](https://englishlive.ef.com/pt-br/blog/false-friends-falsos-cognatos-em-ingles-parece-mas-nao-e/).

As soon as this is fixed, I can merge. Thanks again for your help and sorry for the delay.

Comment (carlosschults) [inline README.asc]:
```suggestion
Você pode gerar os arquivos de e-book manualmente com o Asciidoctor. Se você executar o seguinte, você _pode_ realmente obter arquivos de saída em HTML, Epub, Mobi e PDF:
```

**Comment by carlosschults** [comment]
Any updates on this, @paulolimac ?

**Comment by carlosschults** [comment]
Closing this since it doesn't have any activity since November. We can reopen it if the contributor resumes their work on it.

**Pull request closed without merging**